prow/plugins/label/label.go

```diff
@@ -205,6 +219,24 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 		}
 	}
 
+	for _, statusMatch := range statusMatches {
+		status := strings.TrimSpace(statusMatch[1])
+		sLabel, validStatus := statusMap[status]
+		if validStatus {
+			// true is for testing purposes only, need to figure out way to  mock for testing
+			if true || hasStatusPrivilege(ae.login) {
```

Comment:
@foxish @apelisse @crimsonfaith91 this is a placeholder for the actual logic.  Right now, hasStatusPrivilege has some hardcoded values (the OWNERS_ALIAS url and the key we expect sig-leads to be under).  Suggestions for improving are welcome